### DISCUSSION TITLE
q-table dynamic and selectable width

### DISCUSSION THREAD

**Discussion by tdsereno** [quasarframework/quasar#13622] [Ideas / Proposals] [answered]
Hello, i need a feature that I couldn't find in the Quasar documentation.

The resource consists, in a QTable, the user can change the size of the columns dynamically, as in Google Spreadsheets:

![image](https://user-images.githubusercontent.com/13877439/172212918-772c8621-aca7-4951-8ba1-1ff8bd056dcc.png)
![image](https://user-images.githubusercontent.com/13877439/172213197-4c99bae1-6d14-4aaf-b5a2-936eb3f95db2.png)

For the user it is simple, he passes the mouse over column A and B, and drags, this increases the size of the column

how can i do this in a QTable?

**Comment by MilosPaunovic** [comment, marked as answer]
[https://codepen.io/Hawkeye64/pen/dypyNWL](https://codepen.io/Hawkeye64/pen/dypyNWL)
Credits for that go to Jeff 🙂

**Reply by tdsereno** [reply]
thank you, that's exactly it
